.github/workflows/checkbook_build.yml

```diff
@@ -0,0 +1,40 @@
+name: Checkbook - Build
+
+on:
+  workflow_call:
+    inputs:
+      run_export:
+        type: boolean
+        required: true
+
+env:
+  AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
+
+jobs:
+  build:
+    name: Build Dataset
+    runs-on: ubuntu-22.04
+    container:
+      image: nycplanning/build-geosupport:latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: products/db-checkbook
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
more of a question than feedback, but what is this line doing?

Reply:
it's my understanding that `actions/checkout@v3` is checking out the repo during the github action run so that every step after this has access to the files in the repo at a specific commit, by default the commit from which the run was triggered ([link to this in a build](https://github.com/NYCPlanning/data-engineering/actions/runs/5740463258/job/15558375820#step:3:19), [link to docs](https://github.com/actions/checkout))